apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -184,7 +172,7 @@ If you want to learn more about Prisma, check out the [docs](https://www.prisma.
 
 ## Troubleshooting
 
-### 1. Missing grants
+### Missing grants
 
 If you run `prisma migrate dev --name init` multiple times, it sometimes asks if you want to recreate the whole schema. If you chose yes, it will delete the public schema and recreate it. The default grants are missing after this.
```

Comment:
This is not correct. It will ask you to reset your database if you got your database into a state of drift. That is very different from what is described here.